### DISCUSSION TITLE
No interactive JS components in Tailwind. Yes or No?

### DISCUSSION THREAD

**Discussion by manjeshrv592** [tailwindlabs/tailwindcss#11816] [General]
Hi guys, Am new to tailwind. I have been creating small project to get familiar with tailwind. One thing I noticed is there are no interactive components in tailwind like dropdown, accordion, carousel etc. We have to write our own javascript to get these functionalities. I have been using bootstrap for years and expected tailwind to have same kind of components. Since it's utility framework am assuming there are no interactive components. Please correct me if am wrong I searched through documentation of tailwind and found out there is something called as tailwind UI for components which costs like 300usd. So am I stuck with just static UI with tailwind and have to write my own js to achieve the interactivity.

**Comment by wongjn** [comment]
Yes, you're right. Tailwind is a lower-level library than Bootstrap and abstracts CSS only. You could even build a Bootstrap clone *with* Tailwind!
